lib/Ray/sources/IRessource.hpp

```diff
@@ -0,0 +1,24 @@
+/*
+** EPITECH PROJECT, 2021
+** Bomberman
+** File description:
+** Ressource
+*/
+
+#ifndef RESSOURCE_HPP_
+#define RESSOURCE_HPP_
+
+#include <string>
+
+namespace RAY {
+    class IRessource {
```

Comment:
I believe that an interface that allow one to load something without knowing what it is is not useful at all.

I don't see the point of this interface

Reply:
This may be useful, since we can all unload runtime-loaded-data-based objects at the same time, or manage them all in a container